sql/core/src/main/scala/org/apache/spark/sql/execution/python/PythonArrowOutput.scala

```diff
@@ -33,12 +33,14 @@ import org.apache.spark.sql.vectorized.{ArrowColumnVector, ColumnarBatch, Column
 
 /**
  * A trait that can be mixed-in with [[BasePythonRunner]]. It implements the logic from
- * Python (Arrow) to JVM (ColumnarBatch).
+ * Python (Arrow) to JVM (output type being deserialized from ColumnarBatch).
  */
-private[python] trait PythonArrowOutput { self: BasePythonRunner[_, ColumnarBatch] =>
+private[python] trait PythonArrowOutput[OUT <: AnyRef] { self: BasePythonRunner[_, OUT] =>
```

Comment:
qq: should it be `<: AnyRef`?

Reply:
We assign `null` to the OUT type (although that's a trick) hence need to be AnyRef at least if I understand correctly.